File: test/Data.Analyzers.Vsix.Test/CSharp/DuplicateMounterRegistration.cs
namespace DevZest.Data.Analyzers.Vsix.Test.CSharp
{
    public class DuplicateMounterRegistration : Model
    {
        protected static readonly Mounter<_Int32> _Column1 = RegisterColumn((DuplicateMounterRegistration _) => _.Column1);

        static DuplicateMounterRegistration()
        {
            RegisterColumn((DuplicateMounterRegistration _) => _.Column1);
        }

        public _Int32 Column1 { get; private set; }
    }
}

File: test/Data.Analyzers.Vsix.Test/VisualBasic/DuplicateMounterRegistration.vb
Public Class DuplicateMounterRegistration
    Inherits Model

    Protected Shared ReadOnly _Column1 As Mounter(Of _Int32) = RegisterColumn(Function(x As DuplicateMounterRegistration) x.Column1)

    Shared Sub New()
        RegisterColumn(Function(x As DuplicateMounterRegistration) x.Column1)
    End Sub

    Private m_Column1 As _Int32
    Public Property Column1 As _Int32
        Get
            Return m_Column1
        End Get
        Private Set
            m_Column1 = Value
        End Set
    End Property
End Class
